lib/SIL/IR/SILDeclRef.cpp

```diff
@@ -734,7 +734,7 @@ bool SILDeclRef::isSetter() const {
 }
 
 AbstractFunctionDecl *SILDeclRef::getAbstractFunctionDecl() const {
-  return dyn_cast<AbstractFunctionDecl>(getDecl());
+  return dyn_cast_or_null<AbstractFunctionDecl>(getDecl());
```

Comment:
When can this be null now?

Reply:
I modified `SILDeclRef::getDecl` to return `nullptr` if it doesn't wrap a `ValueDecl`